environments.json

```diff
@@ -1500,6 +1500,20 @@
       "esintl"
     ]
   },
+  "duktape20": {
+    "full": "Duktape 2.0.x",
```

Comment:
I believe you could drop de ".x", it is implicit in the other browsers listed here.